src/System.Net.Security/src/Resources/Strings.resx

```diff
@@ -441,4 +441,7 @@
   <data name="net_nego_server_not_supported" xml:space="preserve">
     <value>Server implementation is not supported</value>
   </data>
+  <data name="net_nego_protection_level_not_supported" xml:space="preserve">
+    <value>Requested protection level is not supported</value>
```

Comment:
@stephentoub This case occurs when a *nix client negotiates Kerberos Sign Only against a windows server. The fix is already in place (in Native layer, part of same PR). However, the fix will work starting MIT Kerberos 1.14 or Heimdal Kerberos implementation on *nix side.

The cause of failure is bit complex to to give out in error message, hence I put a generic statement for the platform not supported exception.
Should I include the error explanation in the error message ?